user/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+class User(models.Model):
+    name              = models.CharField(max_length = 50)
+    mobile            = models.CharField(max_length = 50)
+    email             = models.CharField(max_length = 50, null =True)
```

Comment:
```suggestion
    email             = models.CharField(max_length = 50, null =True, unique=True)
```
유일해야하는 데이터는 로직만이 아니라 모델에서도 검증해주세요.

Reply:
밀리의 서재 사이트에서 자체서비스에 가입했는지, 소셜로그인 유저인지에 따라서도 유저를 분리하는데 
독특하게도 이메일 과 모바일 번호 중복을 허용하고 있습니다. usertype 을 통해 어떤 방식으로 가입한 계정인지 구분할 예정입니다.
User name은 중복 불가라 unique=True 옵션을 추가하였습니다.

Reply:
네 알겠습니다!

Reply:
그렇다면 컬럼중에 소셜 아이디는 어디에 저장하죠?

Reply:
소셜로그인에 대해 아직 이해가 부족하여 가능한지는 모르겠지만 Usertype 테이블에서 번호/카카오/네이버/구글   등으로 나누고 User 테이블에서 usertype을 ForeignKey로가지게 하여 User id를 통해 유저를 구분하려고 하였습니다.

Reply:
User 테이블에 social_id 추가하였습니다.